src/bitmessagecurses/__init__.py

```diff
@@ -264,6 +264,7 @@ def dialogreset(stdscr):
     stdscr.keypad(1)
     curses.curs_set(0)
 def handlech(c, stdscr):
+    addr=""
```

Comment:
Not sure why. Maybe you're trying to fix a possible crash when subscribing the the current address (line 611)? That's a bug there, it should be fixed there.

Reply:
I got error like  VARI accessed before assigment , and this fixed it

Reply:
anyway my other __init__.py has many more fixes , the curses mode is gonna be quite nice soon

Reply:
I only found one possible reason, line 611, which is a bug and should be fixed there.

Reply:
old 611 is    
subscriptions.append([label, addr, True])    
? lines are new now

Reply:
As far as I can see, it should be something like

    subscriptions.append([label, addrbook[abookcur][1], True])

rather than 

    subscriptions.append([label, addr, True])

Analogously in line 614.

That's the only section I found where `addr` is uninitialised.